src/validators/length.ts

```diff
@@ -0,0 +1,25 @@
+export interface LengthArgs {
+  length: number;
+}
+
+export function parseLengthParams(
+  customParams: LengthArgs,
+  errorMessage: string
+) {
+  const length =
+    customParams.length === null ? NaN : Number(customParams.length);
+  if (isNaN(length)) {
```

Comment:
Is this casting Number(customParams.length); necessary?

Reply:
I think is defensive programming if somebody by mistake enter a non numeric length number it just throws an error (was on the original validator I have ported)